### DISCUSSION TITLE
q2 attivazione

### DISCUSSION THREAD

**Discussion by inoguer22** [Mr-MIBonk/M.I.B._More-Incredible-Bash#95] [Q&A]
Zunächst grüße ich alle Mitglieder dieses Forums und wünsche Ihnen ein gutes Jahr 2023.
Ein Freund von mir besitzt einen Audi Q2 mit MHS2_ER_AU_P2035 MU 0235.
wie kann ich als anfänger den navigator entsperren, kann mir jemand helfen.
Grüße von inoger (Gabriele).

**Comment by harman-f** [comment]
Only support for MHI2(Q) unit here!
